src/main/java/gdsc/konkuk/platformcore/controller/event/EventController.java

```diff
@@ -0,0 +1,51 @@
+package gdsc.konkuk.platformcore.controller.event;
+
+import gdsc.konkuk.platformcore.application.attendance.AttendanceService;
+import gdsc.konkuk.platformcore.application.event.EventService;
+import gdsc.konkuk.platformcore.global.responses.SuccessResponse;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.net.URI;
+import java.time.LocalDate;
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/v1/events")
+@RequiredArgsConstructor
+public class EventController {
+  private final EventService eventService;
+  private final AttendanceService attendanceService;
+
+  @GetMapping
+  public ResponseEntity<SuccessResponse> getEventsOfTheMonth(
+      @RequestParam Integer year, @RequestParam Integer month) {
+    List<EventOfMonthResponse> eventOfMonthResponses =
```

Comment:
Response와 관련된 객체 생성을 컨트롤러에서 담당하게하는 것이 `어떨까요?`
- 이미 DTO만 서비스레이어에서 컨트롤러로 넘기고 있는데 Response라는 네이밍이 저는 어떤 데이터가 들어있는지 잘 모르겠어요.

```suggestion
    List<MonthlyEvent> monthlyEventList = eventService.getMonthlyEventList(LocalDate.of(year, month, 1))

```
월별 이벤트를 의미하는 네이밍으로 수정하는것은 어떨까요?

Reply:
넵 그렇게 하는게 좋을 것 같네요! 일급컬렉션을 사용했을때의 장점은 뭘까요? 또 일급 컬렉션을 사용할 시 컬렉션의 불변성을 보장해야하는데 그렇다면 EventOfMonth는 조회만을 위한 로직에서만 사용되는 걸까요?

Reply:
정답은 없는것 같습니다! 개인적으로 일급컬렉션은 네이밍뿐만 아니라 상태와 행위를 한번에 관리할 수 있다는 점이 매력적인 것 같아요. 그래서 도입해도 좋을 것 같다는게 제 생각입니다!

이산님께서 언급하신 DTO의 목적성이 드러난다는것에 동의합니다. 도입해보는것도 좋아보여요~